### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by TheJaredWilcurt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,20 @@
 The public website for the FLIF project.
 
 * **[FLIF.info](http://flif.info)**
+
+* * *
+
+## Contributing
+
+To run the app locally:
```
Comment (jonsneyers) [inline README.md:11]: "app"?

This is only needed for the fancy 404 stuff, if you just want to modify the content you can edit and view the site locally. No need to scare away potential contributors.

Comment (waldyrious) [inline README.md:11]: OMG, yes :/ this definitely is overcomplicating the project (site) for very little benefit IMO.

**Comment by TheJaredWilcurt** [comment]
Sorry, I was in a hurry and half asleep when I wrote this.

Most of the projects I work on are either Single Page Apps (similar to stuff like Gmail or Grooveshark), or actual desktop applications (like [UGUI: FLIF](flif-hub.github.io/UGUI_FLIF)). I just wrote "app" out of habit. Really it should have just said "To run locally:". But this is all meant to be iterative anyways. Create and improve.

Running a simple (50 line) web server to run a site locally isn't that complex of a task. You literally download one thing and then double-click a file (on Windows, in other OS's you type two words). I can even edit the server file to auto open a tab in your browser taking you right to localhost:8000. I didn't add that in, as I thought it might be annoying. Benefits of running locally are that you can see when the site is making failed requests (flagged in red by the local server), which was actually pretty helpful in working on cleaning it up and reorganizing. And more importantly it allows root level linking to run locally.

Stuff like this won't run locally

```
<link rel="shortcut icon" href="/favicon.ico"      type="image/ico" />
<link rel="stylesheet"    href="/_style/style.css" type="text/css"  />
```

As it will be looking for the files here: `file://c:/_style/style.css`. As that is the root of the drive it is being ran on. But running it as a local server means that we can take advantage of the consistency, so that code (which will be used on every page) doesn't ever need to be altered if we put html files in subfolders. It's just future proofing.

The only aspect of increased complication in the new code is modularizing the CSS. But Sass is [very popular and a common aspect of web development](http://i.imgur.com/TKHkLlp.png) with lots of added benefits, most importantly being the automated process of concatenating and minifying the output.

**Comment by jonsneyers** [comment]
Now obsolete, we are going to use Jekyll
(https://help.github.com/articles/using-jekyll-with-pages/)

**Pull request closed without merging**